static/js/dashboard.js

```diff
@@ -331,12 +331,40 @@ function switchFrom(el1, el2, el3, el4) {
 
 var firstSwitch = true
 
+if ($("#proposals-table_wrapper").length === 0) {
+  $(".proposal-switch").css("position", "static")
+}
+
+function waitForTable(selector) {
```

Comment:
Although the solution with the returned Promise looks elegant, it leads to a flickering when you add your first validator (if none was selected before) 

I would suggest a more simple solution: Set the display of the switch initially to 'none' and when we draw the table createProposedChart we also set remove the display none from the switch. 

There is not need to display the switch toggle unless we have data